### PR TITLE
[WIP] Fix #2137: Added SIOP logo in 'Who Uses EvalAI' 

### LINKED ISSUE #2137: Add SIOP logo in "Who uses EvalAI" list.
Here is the SIOP [logo](https://drive.google.com/file/d/1zcw3uqDyM1iHha-o1_zUzKSAkt__98sG/view)

### PR DESCRIPTION
**Pull request by anuj-rathore**
Fix #2137

### CODE REVIEW

**Comment by gautamjajoo** [comment]
@anuj-rathore Please paste the screenshot of the screen in normal view and mobile view as well.

**Comment by anuj-rathore** [comment]
Normal View:
screenshot from 2019-02-20 01-03-48

Mobile View:
screenshot from 2019-02-20 01-03-34

Please have a look.

**Comment by anuj-rathore** [comment]
I have made changes as you instructed me, please have a look!

Here are the screenshots to compare:
Earlier:
screenshot from 2019-02-20 20-03-35

screenshot from 2019-02-20 20-04-18

screenshot from 2019-02-20 20-03-44


Now:
screenshot from 2019-02-20 20-04-42
screenshot from 2019-02-20 20-03-09
screenshot from 2019-02-20 20-02-45

File: frontend/src/views/web/landing.html [hunk 1]
```diff
@@ -161,40 +162,53 @@ <h4><div>Who uses EvalAI?</div></h4>
...
                             <div class="col s12 l3">
                                     <div class="org-logo">
+                                        <br>
                                         <img src="dist/images/organizations/stanford.png">
                                     </div>
                                </div>
                             <div class="col s12 l3">
                                     <div class="org-logo">
-                                        <img src="dist/images/organizations/adelaide.png" id="adelaide-logo">
+                                        <br>
```
Comment (gautamjajoo) [inline frontend/src/views/web/landing.html]: If you are removing css tags, remove the code in css file too!

File: frontend/src/views/web/landing.html [hunk 2]
```diff
@@ -134,19 +134,20 @@ <h4><div>Who uses EvalAI?</div></h4>
                                </div>
```
Comment (RishabhJain2018) [inline frontend/src/views/web/landing.html]: @anuj-rathore Why are you removing the `id` and why fb logo file is added again?

**Comment by RishabhJain2018** [comment]
Closing this since it already merged now!

**Pull request closed without merging**